raiden/routing.py

```diff
@@ -69,7 +65,9 @@ def get_best_routes(
             if is_usable is channel.ChannelUsability.USABLE:
                 address_to_address_metadata = {Address(from_address): our_address_metadata}
                 try:
-                    address_metadata = query_address_metadata(pfs_config, to_address)
+                    address_metadata = None
+                    if pfs_config is not None:
+                        address_metadata = query_address_metadata(pfs_config, to_address)
                     if address_metadata:
```

Comment:
It is actually not necessary. That is the reason why I did not change it in every place in the tests. It will have two different outcomes. 

With route states: It has mocked address metadata and will not call `get_best_routes`
Without route states: It will call `get_best_routes` and will not query address metadata as `pfs_config is None` and leave the address_metadata field for its partner empty.